### DISCUSSION TITLE
Submission for contest from MIPLIB

### DISCUSSION THREAD

**Discussion by Dr-Irv** [ambros-gleixner/MIPcc23#4] [General]
I'm not sure that what I submitted to MIPLIB a few years ago will meet your criteria.

See https://miplib.zib.de/instance_details_seqsolve1.html for a description, followed by https://miplib.zib.de/instance_details_seqsolve2short4288.html and then https://miplib.zib.de/instance_details_seqsolve3short4288excess384.html

These are 3 problems, where hierarchical optimization is used.  So it is valid to take a solution from the first problem and use it to create a MIP start for the second problem, and then take a solution from the second problem and use it as a MIP start for the third problem.

**Comment by sbolusani** [comment]
Thank you for the suggestion. I sent you a private email to continue this discussion offline. Please let me know if you did not receive any such email.
